Makefile

```diff
@@ -95,79 +98,87 @@ help:
 	@echo -e " \033[1mCLEANING TARGETS\033[0m "
 	@echo "- clean              Clean genereated files"
 	@echo "- clean_venv         Clean python venv"
-	@echo -e " \033[1mDJANGO TARGETS\033[0m "
-	@echo "- django-check       Inspect the Django project for common problems"
-	@echo "- django-migrate     Synchronize the database state with the current set of models and migrations"
+	# @echo -e " \033[1mDJANGO TARGETS\033[0m "
+	# @echo "- django-check       Inspect the Django project for common problems"
+	# @echo "- django-migrate     Synchronize the database state with the current set of models and migrations"
 
 
 # Build targets. Calling setup is all that is needed for the local files to be installed as needed.
 
-.PHONY: dev
-dev: $(DEV_REQUIREMENTS_TIMESTAMP)
+# .PHONY: dev
+# dev: $(REQUIREMENTS_DEV_TIMESTAMP)
 
 
 .PHONY: setup
-setup: $(REQUIREMENTS_TIMESTAMP)
-
+setup:
+	test -d $(VENV) || ( $(SYSTEM_PYTHON) -m venv $(VENV) && $(PIP) install --upgrade pip setuptools; \
+	$(PIP) install -U pip wheel; \
+	$(PIP) install -r $(REQUIREMENTS_DEV); )
+	test -d app/config/settings.py || echo "from .settings_dev import *" > app/config/settings.py
```

Comment:
The issue with this is that if we change the `requirement.txt` file (also remotely changed by a git pull) the requirements are not updated unless we run `make clean setup` of manually call `pip install`. In my opinion using here the timestamp way takes advantage of the makefile syntax and is more appropriate. It is also more user friendly

Reply:
I changed it such that the `pip install -r requirements_dev.txt` runs every time you execute `make setup` (so e.g after a `git pull`)